### ISSUE TITLE
MessageBird returning !200s still makes Sachet return 200s

### ISSUE THREAD

**Issue by EdSchouten** [marcelcorso/sachet#5]
Quick bug report. I should likely fix this myself and send a pull request, but ENOTIME, so I'm quickly making sure this won't get lost.

Looking at `messagebird.go`, there is a TODO that there is no support for error checking. It would be nice if this printed the HTTP error message in the logs. Even if we're too lazy to do that, we should make sure that we're not reporting HTTP 200 back to AlertManager, as that makes AlertManager assume that the SMS was sent correctly.

To summarise: we should patch up `messagebird.go` to return an error. `main.go` should return, say, a HTTP 500 when an error has occurred.
